sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md

```diff
@@ -1,33 +1,36 @@
+The authentication events trigger for Azure Functions allows you to implement a custom extension to handle Azure Active Directory (Azure AD) authentication events.  The authentication events trigger handles all the backend processing for incoming HTTP requests for Azure AD authentication events and provides the developer with:
 
-This project framework provides the following features:
-
-* Token validation for securing the API call
-* Object model, typing, and IDE intellisense
-* Inbound and outbound validation of the API request and response schemas
-* Versioning
-* No need for boilerplate code.
+- Token validation for securing the API call
+- Object model, typing, and IDE intellisense
+- Inbound and outbound validation of the API request and response schemas
+- Versioning
 
 ## Getting started
 
 ### Install the package
 
+Install the authentication events trigger for Azure Functions with [NuGet](https://www.nuget.org/):
+
 ```dotnetcli
 dotnet add package Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents --prerelease
 ```
 
 ### Prerequisites
 
-* [Visual Studio 2019](https://visualstudio.microsoft.com/vs/) for Windows **OR** [Visual Studio Code >= 1.61](https://code.visualstudio.com/download)
-* [Dotnet core 3.1](https://dotnet.microsoft.com/download/dotnet/3.1)
-* [Azure function tools 3.30](https://github.com/Azure/azure-functions-core-tools)
-* [Nuget](https://docs.microsoft.com/nuget/install-nuget-client-tools)
-* [Azure Function Core Tools](https://github.com/Azure/azure-functions-core-tools#installing)
-* If using Visual Studio Code the following extensions:
-  * [ms-azuretools.vscode-azurefunctions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions)
-  * [ms-dotnettools.csharp](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp)
+- **Azure Subscription:**  To use Azure services, including Azure Functions, you'll need a subscription.  If you do not have an existing Azure account, you may sign up for a [free trial](https://azure.microsoft.com/free/dotnet/) or use your [Visual Studio Subscription](https://visualstudio.microsoft.com/subscriptions/) benefits when you [create an account](https://account.windowsazure.com/Home/Index).
+- [Visual Studio 2019](https://visualstudio.microsoft.com/vs/) or later for Windows **OR** [Visual Studio Code >= 1.61](https://code.visualstudio.com/download) or later
+- [Dotnet core 3.1](https://dotnet.microsoft.com/download/dotnet/3.1)
+- [Azure function tools 3.30](https://github.com/Azure/azure-functions-core-tools) or later
+- [Nuget](https://learn.microsoft.com/en-us/nuget/install-nuget-client-tools)
```

Comment:
Can you help me understand why this is listed?  Generally, this is not something we include.

Reply:
Removing most of these prereqs, they should be taken care when installing the package.